test/unit/index.js

```diff
@@ -9,5 +12,5 @@ testsContext.keys().forEach(testsContext)
 // require all src files except main.js for coverage.
 // you can also change this to match only the subset of files that
 // you want coverage for.
-const srcContext = require.context('../../src', true, /^\.\/(?!main(\.js)?$)/)
+const srcContext = require.context('../../src/components', true, /^\.\/(?!main(\.js)?$)/)
```

Comment:
what about the util folder, could we solve this by excluding the form demo ? or some other means